ui/src/app/shared/components/page/page.scss

```diff
@@ -48,30 +48,29 @@
     }
 }
 
-.login-logout-button {
-    border-radius: 3px;
-    @include themify($themes) {
-        color: themed('light-argo-teal-7');
-    }
-    padding: 3px;
-    cursor: pointer;
-
-    &:focus {
-        color: $argo-color-teal-5;
-        box-shadow: 0 0 0 0.1em $argo-color-teal-5;
-    }
-    &:hover {
-        @include themify($themes) {
-            color: themed('light-argo-teal-5');
-        }
-    }
-}
-
 .page {
     padding-left: 0 !important;
     &__top-bar {
         left: $sidebar-width !important;
     }
+
+    &__button {
+        border-radius: 5px;
+        color: white;
+        padding: 7px 14px;
+        cursor: pointer;
+        font-size: 14px;
+        font-weight: 600;
+        text-align: center;
+        background-color: $argo-color-gray-6;
+        &:hover {
+            background-color: $argo-success-color;
+        }
+
+        .fa-right-to-bracket {
+            margin-right: 3px;
+        }
+    }
```

Comment:
this new button design seems reusable if planning to apply for all buttons in future. So either in argo-ui with appropriate class-name or somewhere like new `override.scss` file in this repo. What are your thoughts on it?

My take is argo-ui is more suitable place, we know what type of buttons could exist thus will be easy to not to miss any state (disabled, hover, focus etc.) plus easier to find.

Reply:
Just to add to @Marvin9 's comment, the buttons in the sliding panels (eg. the New App sliding panel, as an example) will have to be changed.  And would these redesigned buttons affect the redesign of the sliding panels that Kostis (?) was working on?

Reply:
@Marvin9 I agree that this will be applicable to all buttons in the future, and that theoretically argo-ui is the best place to put this new style. And @keithchong, you're absolutely right that the buttons in all those places will need to be updated (not sure about the sliding panel redesign, it's been quite a while since I heard any updates on that). That being said, in practice I made this change with a few things in mind:

1. By updating the buttons piecemeal, it reduces the "blast radius" and review burden for the PR (granted it's just buttons, so the potential for damage is low). Principally I feel strongly that an interface should have cohesion, but practically in an open source project such as Argo CD it's difficult to make progress if we don't go step by step. 
2. Similar to above, updating `argo-ui` has implications for Argo Workflows. While it's an option to add a _new_ button classname to `argo-ui`, at that point it would only be consumed by Argo CD and thus should stay in Argo CD. This is a topic for another day, but as the Workflows and CD UIs diverge I see less and less upside to the shared repo; it's lengthens the dev cycle to merge changes in `argo-ui` first, and if those changes are only used by one repo it's for naught.

Reply:
Alright thanks for explaining your thoughts, as long as you are clear we good. Would you just add definition for other variant used in top-bar? eg. Disabled button.

Just asking, what do you think about the reaction from users of different button style in top-bar and everywhere else?

Reply:
I agree with the changes in stages. Because of this, not quite sure if things are intentionally missed, to work on later, or perhaps unintentionally missed, in which case, it's probably better to just ask the question in the PR review, to clarify the intent and direction. So, I think this LGTM.

Also, all the staged changes will need to be done before the next release so there should be minimal (or zero) discrepancies/differences in look and feel.

Reply:
@Marvin9 Yeah, I'll add another variant for disabled buttons.

@keithchong Hopefully this answers your question, I only scoped these changes to the top bar, so if there is anything missed in the top bar let me know (I'm pretty confident I got all of them since there are no hidden buttons 🙂).

I agree, I think given that 2.8 is about to be released and this will not be included, we will have maximum amount of time to migrate all buttons before 2.9